src/main/filter.ts

```diff
@@ -9,7 +9,7 @@ type FilterMap = { [onestopFilterName: string]: FilterFunction };
 const BROS_GENDER: string = "Male";
 const SIS_GENDER: string = "Female";
 // Maps a filter's name on the Onestop to its corresponding filter function
-const FILTER_MAP: FilterMap = {
+export const FILTER_MAP: FilterMap = {
```

Comment:
```suggestion
const FILTER_MAP: FilterMap = {
```
Does not look like this is being used outside of this module so it does need to be exported